torch/onnx/symbolic_helper.py

```diff
@@ -712,6 +712,16 @@ def _index_fill_reshape_helper(g, self, dim, index):
     expanded_index = expand(g, unsqueezed_index, expanded_index_shape, None)
     return expanded_index_shape, expanded_index
 
+# When using reshape helper (opset_version >= 14), if reshape has -1,
```

Comment:
Will this be relaxed to not allowing both -1 and 0 appearing together in reshape after Ashwini's fix?

Reply:
Yes essentially, once Ashwinis fix is in, I will relax this condition and get rid of the workaround